lua-aws/engines/fs/msys-posix.lua

```diff
@@ -0,0 +1,17 @@
+-- Copyright (c) 2016 Message Systems, Inc. All rights reserved
+
+local posix = require("posix")
+
+return {
+  scandir = function(path, prefix, fn)
+    local pattern = "^" .. prefix
+    local d = posix.dir(path)
+    for _, file in ipairs(d) do
+      if file:match(pattern) then
+        fn(path .. "/" .. file)
```

Comment:
Looking at engines/fs/lfs.lua and it seems there they assume path already ends with a "/".  So are we ending up with double "/" here? (which shouldn't matter- just thought I'd point it out).